jobs/rocoto/postanl.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash -x
```

Comment:
Curious on how this differs from `post.sh` other than the `exit 99` for `FHRGRP` that is for the forecast hours?

Reply:
https://github.com/NOAA-EMC/global-workflow/blob/develop/jobs/rocoto/post.sh

Reply:
You are correct.  The only difference is the lines you identify.   The UPP team should take the lead on this change.   The UPP application for the analysis differs from the forecast.   

We run executable `gfs_ncep_post -> ../sorc/upp.fd/exec/upp.x` when processing the `atmanl` and `sfcanl` files.   `upp.x` output is then used to generate analysis pgrb files.   The model runs the inline UPP.   `post.sh` does not execute `upp.x`  Instead, it processes inline UPP output to create pgrb files.  When `post.sh` includes the analysis files, the post job which processes analysis files and possibly initial forecast hours has different dependencies than subsequent post jobs which only process inline UPP output from the forecast model.   'postanl.sh` was added in recognition of these distinctions.   

Initial cycling tests with JEDI-VAR benefited from moving the analysis UPP into its own job.   Subsequent local changes to JEDI-VAR _may_ lessen the need for this.   This would need to be tested to confirm.   

That said, the above distinctions between analysis and forecast UPP applications remain.  If the UPP team were to adopt this approach, you are right in that `postanl.sh` could and should be simplified.

Reply:
If the scripts are identical, we should just use the same script. I definitely don't think it is worth it to maintain all this duplicate code for a single variable bounds check. If we really think it is necessary (I don't), we can put it in the config file.

Reply:
'config.postanl' and `postanl.sh` have been removed from a working copy of `feature/ufsda-gdasapp` on Hera.   References to `postanl` have been removed from `setup_workflow.py`.  A four cycle GSI-based DA parallel is now running on Hera to confirm no change in the results.    A four cycle JEDI-based DA parallel will be run on Orion once the machine is returned to service.   Assuming both tests pass, `postanl` will be removed from the branch.

Reply:
[6674720](https://github.com/NOAA-EMC/global-workflow/commit/667472055b8db74e05467f0c7b02eb6a67bce9d1) removes all references to postanl.   This includes `jobs/rocoto/postanl.sh`, `parm/config/config.postanl`, `postanl` sections in `config.resources`, and `postanl` sections in `ush/rocoto/setup_workflow.py`